ecl/sss/v2/_proxy.py

```diff
@@ -543,3 +544,17 @@ def delete_workspace_role_assignment(self, workspace_id, user_id):
         return workspace_role.delete_workspace_role_assignment(
             session=self.session, workspace_id=workspace_id, user_id=user_id
         )
+
+    def get_user_token_by_admin(self, user_id, tenant_id=None, no_catalog=True):
+        """Obtain a user token on behalf of the user
```

Comment:
```suggestion
        """Obtain a user token by SSS on behalf of SDP GUI
```
瑣末なことですが、代理での取得とは「これまでKeystone tokenは SDP GUIが取得していたが、SSSが代わりに取得してくれるようになった」という意味かと思います。ユーザーの代わりに取得するというのは違和感を覚えました。

Reply:
@sugimo-s 
ECL SDK は SDP GUI以外からも実行可能なので ` of SDP GUI` と言う文言は避けたいです。
「Obtain a user token by SSS on behalf」 (SSSによるユーザートークンの取得代行) ではいかがでしょうか。